source/stylesheets/site.css

```diff
@@ -119,6 +119,21 @@ td.browser-column {
   the credits
 */
 
+.heart-circle {
+  width: 72px;
+  height: 72px;
+  border-radius: 50px;
+  background-color: white;
+  margin: auto;
+  box-shadow: 0 1px 2px 0px rgba(0, 0, 0, 0.15);
```

Comment:
matched it with the top bar.  guess that's wrong too - you should tell whoever did that to change it.

done